### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by yaroslav-furhala**
- [DEMO LINK](https://yaroslav-furhala.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://yaroslav-furhala.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,9 +8,39 @@
...
+    </style>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+
+      <label class="formfield">
```
Comment (ivanchuk-yurii) [inline src/index.html:21]:
```suggestion
      <label class="form-field">
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  display: block;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+.form {
+  display: block;
+  margin-bottom: 20px;
+}
+#site-search-big {
+  background-image: url(https://raw.githubusercontent.com/yaroslav-furhala/layout_search-bar-airbnb/2bf458ddfbb9ed1bb6beca1e6040ee3e75d5df10/src/images/Search.svg);
```
Comment (ivanchuk-yurii) [inline src/style.css:15]: download this file and use from your directory

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  background-position: 10px center;
+  padding-left: 50px;
+  border-radius: 5px;
+  background-size: 2em;
+  background-repeat: no-repeat;
```
Comment (ivanchuk-yurii) [inline src/style.css:20]: combine all this styles into one `background` shorthand

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+}
+#site-search-big:hover {
+  background-color: #ff0;
+}
+#site-search-small:focus {
+  background-color: #f00;
+}
+#site-search-small:hover {
+  background-color: #ff0;
+}
```
Comment (ivanchuk-yurii) [inline src/style.css:47]: write all of this though comma
```suggestion
#site-search-big:focus,
#site-search-big:hover,
#site-search-small:focus,
#site-search-small:hover {
  background-color: #ff0;
}
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,47 @@
+@font-face {
+  font-family: sans-serif;
```
Comment (ivanchuk-yurii) [inline src/style.css:2]: name font-family, Avenir or Avenir-Book

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Task is not ready for review 
expected: image
actual: 
image
If you have any questions feel free to ask in Slack

**Pull request closed without merging**